packages/react-components/src/components/ParticipantsButton.tsx

```diff
@@ -300,6 +300,9 @@ export const ParticipantsButton = (props: ParticipantsButtonProps): JSX.Element
                 maxWidth: '100%'
               }
             },
+            style: {
```

Comment:
This properly sets the maxHeight of the people context menu to 20rem and when we scroll we won't get odd visual artifacts.